google/cloud/storage/client.py

```diff
@@ -48,15 +50,16 @@
 from google.cloud.storage.bucket import Bucket, _item_to_blob, _blobs_page_start
 from google.cloud.storage.blob import (
     Blob,
-    _get_encryption_headers,
+    # _get_encryption_headers,
```

Comment:
Before merging, be sure to remove this comment and the above comments. In general, prefer to delete rather than to comment out - with git, it's easy to restore whatever was there earlier even if you delete it.